### PR TITLE
code challenge submission

### PR DESCRIPTION
**Pull request by rckchen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +32,7 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
+    "css-loader": "^0.28.11",
```
Comment (mattiaerre) [inline package.json]: this is cool, I think we should add it by default to the challenge, what do you think @scottlepich ?

File: src/client/app/App.js [hunk 2]
```diff
@@ -1,7 +1,59 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-const App = () => (
-  <h2>Stats Widget</h2>
-);
+import './app.css';
+
+class App extends Component {
+  static priceRanges(prices) {
```
Comment (mattiaerre) [inline src/client/app/App.js]: uhm, interesting. not a big fan of static methods to be honest.

File: src/client/app/App.js [hunk 3]
```diff
@@ -1,7 +1,59 @@
...
+    const { prices } = this.state;
+    let dollarString = '';
+
+    return (
+      <div className="app-container">
+        <h2>Statistics</h2>
+        <div className="stats-container">
+          {
+            prices.map((price) => {
+              dollarString += '$';
```
Comment (mattiaerre) [inline src/client/app/App.js]: I would have kept the dollar string along w/ the data structure.

File: webpack.config.js [hunk 4]
```diff
@@ -16,6 +16,11 @@ const config = {
         test: /\.jsx?/,
         include: APP_DIR,
         loader: 'babel-loader'
+      },
+      {
+        test: /\.css?/,
```
Comment (mattiaerre) [inline webpack.config.js]: awesome, thanks for that @rckchen

Comment (rckchen) [reply webpack.config.js]: Welcome! However this isn't a working product as we also need style-loader as well 
https://github.com/webpack-contrib/style-loader

File: src/client/app/App.js (hunk 2 above)
Comment (rckchen) [reply src/client/app/App.js]: the linter notes to use static method when there is no `this` keyword in the class method which is my reason for doing this.

**Pull request closed without merging**